00_download_data/nbconverted/0_download_data.py

```diff
@@ -32,7 +32,8 @@
 # In[3]:
 
 
-os.makedirs(cfg.raw_data_dir, exist_ok=True)
+if not os.path.exists(cfg.raw_data_dir):
+    os.makedirs(cfg.raw_data_dir)
```

Comment:
For a future PR maybe, you can also migrate this to `pathlib`.

Also, did the `exist_ok` failed for some reason that you are changing it here?

Reply:
No, `exist_ok` should be fine - I was just copying old code. I'll update these.

And yeah, I should really update this to `pathlib` in a future PR.